Code/Framework/AzCore/Tests/Serialization.cpp

```diff
@@ -5712,6 +5712,39 @@ namespace UnitTest
         EXPECT_FLOAT_EQ(4.0f, value);
     }
 
+    TEST_F(Serialization, AttributeInvocable_UsingVoidPointerInstance_Succeeds)
+    {
+        constexpr AZ::Crc32 invokableCrc = AZ_CRC_CE("Invokable");
```

Comment:
`constexpr` variables are still stack variables, there's nothing here that is forcing this to be evaluated in a `constexpr` context (like a `static_assert`). This should be `static constexpr` instead.

Reply:
No actually, this variable is being evaluated compile time by MSVC, GCC and Clang due to the `constexpr` keyword, now it is being placed on the stack however, because there is no const.
https://godbolt.org/z/GqjY44KMM